### DISCUSSION TITLE
Multiple layers config that uses Miryoku layout

### DISCUSSION THREAD

**Discussion by sturmenta** [yqrashawn/GokuRakuJoudo#242] [Show and tell]
# I want to share this layouts-config in case it is useful to anyone.

[the repo with the goku config file](https://github.com/sturmenta/goku-karabiner-miryoku/tree/main)

---

## Goku Karabiner config that uses Miryoku layout

This config uses [GokuRakuJoudo](https://github.com/yqrashawn/GokuRakuJoudo) to create a config JSON file for [Karabiner-Elements](https://github.com/pqrs-org/Karabiner-Elements) *"A powerful and stable keyboard customizer for macOS."* using [Miryoku keyboard layout](https://github.com/manna-harbour/miryoku) with QUERTY layout.

### Miryoku layout (all layers):

>
> Miryoku is an ergonomic, minimal, orthogonal, and universal keyboard layout.
>
<img width="689" alt="1" src="https://github.com/sturmenta/goku-karabiner-miryoku/assets/30802967/f4bbe805-7c40-4c51-aa9d-819bade55e5d">

### The keyboard i'm using: (Keychron K2 v2)
>
> This is the keyboard i'm using, it's a 75% keyboard with a lot of keys, it's a great keyboard, but it's not programmable, so i'm using Goku to create a config for Karabiner-Elements to make it programmable.
>
<img width="1148" alt="2" src="https://github.com/sturmenta/karabiner-config/assets/30802967/2f27b325-14e9-4055-8a47-f39f2b1d8166">
<img width="1059" alt="3" src="https://github.com/sturmenta/karabiner-config/assets/30802967/4a0a328a-ac38-4097-b3c7-1d432867809e">

---
### The mapping i'm using:
```edn
;; ───────────────────────────────────────────────────────────────────
;;
;;      from -- original keymap (my selection of keys used)
;;
;;      1   2   3   4   5       8   9   0   -   =
;;      q   w   e   r   t       u   i   o   p   [
;;      a   s   d   f   g       j   k   l   ;   '
;;              x   c   v       m   ,   .
;;
;; ───────────────────────────────────────────────────────────────────
;;
;;      to -- mapped keymap
;;
;;      q   w   e   r   t       y   u   i   o   p
;;      a   s   d   f   g       h   j   k   l   '
;;      z   x   c   v   b       n   m   ,   .   /
;;              esc spc tab     rtn del supr
;;
;; ───────────────────────────────────────────────────────────────────
```

**Comment by highly-illiquid** [comment]
This looks rad! Any updates or complaints now that you've had some time with this layout?
